Makefile

```diff
@@ -14,7 +14,7 @@ endif
 endif
 
 BUILD_DATE=${shell date +%Y%m%d.%H%M}
-VERSION=v1.1.0
+VERSION=${shell git describe --abbrev=0 --tag}
```

Comment:
I'm sorry, but I would like to leave this line with a static version input like https://github.com/torvalds/linux/commit/136057256686de39cc3a07c2e39ef6bc43003ff6 to know the version in case of without git log.